neurox/data/extraction/transformers_extractor.py

```diff
@@ -103,10 +103,12 @@ def aggregate_repr(state, start, end, aggregation):
         Matrix of size [NUM_LAYERS x LAYER_DIM]
     """
     if end < start:
-        sys.stderr.write("WARNING: An empty slice of tokens was encountered. " +
-            "This probably implies a special unicode character or text " +
-            "encoding issue in your original data that was dropped by the " +
-            "transformer model's tokenizer.\n")
+        sys.stderr.write(
```

Comment:
As mentioned in #15, I'll run the formatting after these PRs are merged, so feel free to revert these lines back so that only relevant code is part of the PR. You don't have to strictly do this however!